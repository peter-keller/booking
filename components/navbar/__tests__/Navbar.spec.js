import { shallowMount } from '@vue/test-utils'

import Navbar from '..'

describe('Component: Navbar', () => {
  const factory = (opts = {}) => shallowMount(Navbar, {
    ...opts
  })

  it('matches snapshot', () => {
    const wrapper = factory()

    expect(wrapper.html()).toMatchSnapshot()
  })
})
