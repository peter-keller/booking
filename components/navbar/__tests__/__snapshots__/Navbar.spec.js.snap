// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: Navbar matches snapshot 1`] = `
"<div class=\\"navbar\\">
  <div class=\\"navbar__inner\\">
    <div class=\\"navbar__top\\">
      <nuxt-link-stub to=\\"/\\">
        <icon-stub name=\\"Logo\\" size=\\"medium\\" stroke=\\"\\" class=\\"navbar__icon\\"></icon-stub>
      </nuxt-link-stub>
      <div class=\\"navbar__user\\">
        <usergreeting-stub></usergreeting-stub>
      </div>
    </div>
    <div class=\\"navbar__search\\"><input type=\\"text\\" placeholder=\\"HOMEPAGE_SEARCH_PLACEHOLDER\\"></div>
    <nuxt-link-stub to=\\"/\\"><img src=\\"../../assets/images/azam.jpg\\" class=\\"navbar__img\\"></nuxt-link-stub>
  </div>
</div>"
`;
